src/modules/linagora.esn.graceperiod/frontend/js/app.js

```diff
@@ -1,5 +1,15 @@
-(function() {
+(function(angular) {
   'use strict';
 
-  angular.module('linagora.esn.graceperiod', ['restangular', 'esn.websocket', 'esn.notification', 'esn.http', 'esn.lodash-wrapper']);
-})();
+  angular.module('linagora.esn.graceperiod', ['esn.websocket', 'esn.notification', 'esn.http', 'esn.i18n']);
+})(angular);
+
+require('../../../../frontend/js/modules/websocket.js');
+require('../../../../frontend/js/modules/notification.js');
+require('../../../../frontend/js/modules/http.js');
+require('../../../../frontend/js/modules/i18n/i18n-string.service.js');
```

Comment:
Isn't it `../../../../frontend/js/modules/i18n/i18n.module.js` ?

Reply:
thanks